tests/post/features/operator.feature

```diff
@@ -0,0 +1,13 @@
+@post @ci @local @operator
+Feature: Operator
+    Scenario: Creation of extra ClusterConfig
+        Given the Kubernetes API is available
+        And pods with label 'app.kubernetes.io/name=metalk8s-operator' are 'Ready'
+        When we create an extra 'test-cc' ClusterConfig
+        Then the 'test-cc' ClusterConfig get deleted
+
+    Scenario: Deletion of the default ClusterConfig
+        Given the Kubernetes API is available
+        And pods with label 'app.kubernetes.io/name=metalk8s-operator' are 'Ready'
+        When we delete the 'default' ClusterConfig
+        Then the 'default' ClusterConfig get automatically recreated
```

Comment:
I'm not sure to get your point here.

I agree that such operator could be used outside of "metalk8s core" at some point, but I don't get why this approach could be an issue for this ?

NOTE: This test will be updated at some point when we will have a proper Webhook to "forbib" object creation and deletion

Reply:
Are you suggesting we shouldn't use a `ClusterConfig` object when running in other K8s distributions @JBWatenbergScality? It's debatable, and adding support for this could be done later IMO.